### PR TITLE
feat: adding new icon (DescriptionOutlinedIcon)

### PR DESCRIPTION
**Pull request by leticiabiazioli**
## Description
Adding DescriptionOutlinedIcon

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [x] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review
- [ ] TypeScript updated

### A11y review
- [ ] A11y checks
  - [ ] accessible via keyboard?
  - [ ] Identifiable through assistive technology (screen-reader chrome extension)?
  - [ ] Semantically correct html elements (or if necessary identified by WAI-ARIA)?
  - [ ] Is there a deficiency in color contrast?

### Browsers review
- [ ] Layout review
  - [ ] Edge
  - [ ] Firefox
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile
- [ ] Ux review validation

### CODE REVIEW

**Review by alizeleal — APPROVED** [review]

File: components/Icon/Icon.jsx [hunk 1]
```diff
@@ -39,6 +39,7 @@ import Contrast from '@mui/icons-material/Contrast';
 import CreditCard from '@mui/icons-material/CreditCard';
 import DateRange from '@mui/icons-material/DateRange';
 import Delete from '@mui/icons-material/Delete';
+import DescriptionOutlinedIcon from '@mui/icons-material/DescriptionOutlined';
```
Comment (MarcosViniciusPC) [inline components/Icon/Icon.jsx:42]:
```suggestion
import DescriptionOutlined from '@mui/icons-material/DescriptionOutlined';
```

File: components/Icon/Icon.jsx [hunk 2]
```diff
@@ -165,6 +166,7 @@ const Icon = ({ name, skin = '', style = {}, size = 'medium', ...props }) => {
     credit_card: CreditCard,
     date_range: DateRange,
     delete: Delete,
+    description_outline: DescriptionOutlinedIcon,
```
Comment (MarcosViniciusPC) [inline components/Icon/Icon.jsx:169]:
```suggestion
    description_outlined: DescriptionOutlined,
```

File: components/shared/icons.js [hunk 3]
```diff
@@ -37,6 +37,7 @@ const icons = [
   'credit_card',
   'date_range',
   'delete',
+  'description_outline',
```
Comment (MarcosViniciusPC) [inline components/shared/icons.js:40]:
```suggestion
  'description_outlined',
```

File: components/Icon/Icon.jsx (hunk 1 above)
Comment (MarcosViniciusPC) [reply components/Icon/Icon.jsx:42]: I believe that the "Icon" prefix is ​​not necessary, it is somewhat redundant. How to improve, I suggest removing it

File: components/Icon/Icon.jsx (hunk 2 above)
Comment (MarcosViniciusPC) [reply components/Icon/Icon.jsx:169]: What do you think about adjusting the name to "outlined"? This way we can keep the nomenclature in accordance with the name of the icon and the other icons already present.

**Review by MarcosViniciusPC — CHANGES_REQUESTED** [review]
[ ] Code review

Add new icon to components/Icon/index.d.ts

**Review by MarcosViniciusPC — APPROVED** [review]
[x] Code review

**Pull request merged**